variables.tf

```
@@ -29,6 +29,12 @@ variable "kind" {
   }
 }
 
+variable "active_directory_tenant_auth_endpoint" {
+  description = "Endpoint for the Entra ID authentication endpoint"
```

Update description to be more similar to other variables:

```suggestion
  description = "The endpoint of the Azure AD tenant use for authentication."
```

And since we still refer to Azure AD instead of Entra ID in other variable descriptions, do so here as well to stay consistent. Can be changed in separate refactor PR.